packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -1,6 +1,7 @@
 import * as async from 'async';
 import * as crypto from 'crypto'
 import * as _ from 'lodash';
+import * as moralis from 'moralis';
```

Comment:
change to `import Moralis from 'moralis';` and remove `const Moralis = moralis.default;` below

Reply:
done